sklearn/preprocessing/discretization.py

```diff
@@ -0,0 +1,261 @@
+# Author: Henry Lin <hlin117@gmail.com>
+
+from __future__ import division, absolute_import
+
+import numbers
+import numpy as np
+import warnings
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.preprocessing.data import _transform_selected
+from sklearn.utils.validation import (
+    check_array,
+    check_is_fitted,
+    column_or_1d,
+    FLOAT_DTYPES
+)
```

Comment:
I don't think compactness is a virtue, but if we're going to have each on a separate line, we might as well have separate import statements